syntax = "proto3";

package service2;

service ProductService {
  rpc GetProduct (ProductRequest) returns (ProductResponse);
  rpc CreateProduct (CreateProductRequest) returns (ProductResponse);
}

message ProductRequest {
  string product_id = 1;
}

message CreateProductRequest {
  string name = 1;
  float price = 2;
  string description = 3;
}

message ProductResponse {
  string product_id = 1;
  string name = 2;
  float price = 3;
  string description = 4;
}

